node/src/rpc/rpc_effects.rs

```diff
@@ -279,6 +279,17 @@ pub fn rpc_effects<S: Service>(store: &mut Store<S>, action: RpcActionWithMeta)
             }
         }
         RpcAction::P2pConnectionIncomingPending { .. } => {}
+        RpcAction::P2pConnectionIncomingAnswerReady {
```

Comment:
This is a case of a new action that is required only because you need to dispatch two different actions from the callback?

Reply:
Yes, but I can remove it, set callback to `RpcAction::P2pConnectionIncomingRespond` and dispatch `P2pConnectionIncomingAction::AnswerSendSuccess` after callback in p2p crate

Reply:
No, it is fine, was just asking to make sure I was interpreting it correctly.